opengapps-packages.mk

```diff
@@ -149,6 +188,8 @@ PRODUCT_PACKAGES += \
     WebViewGoogle
 endif
 
+ifneq ($(filter $(TARGET_GAPPS_VARIANT),pico),) # Not on tvstock
```

Comment:
here, too.

also, is the comment accurate?  how is "pico" related to "tvstock"?

Reply:
tv stock does not include pico variant. Everything else include pico.